src/components/dashboard/AppCard.vue

```diff
@@ -17,6 +17,7 @@ import { useMainStore } from '~/stores/main'
 const props = defineProps<{
   app: Database['public']['Tables']['apps']['Row']
   channel: string
+  deleteButton: boolean
```

Comment:
can we use something more descriptive?

```suggestion
  enableDeleteButton: boolean
```
or something like that